torch_geometric/nn/models/mlp.py

```diff
@@ -111,14 +115,34 @@ def __init__(
         assert len(channel_list) >= 2
         self.channel_list = channel_list
 
-        self.dropout = dropout
+        if isinstance(dropout, float):
+            self.dropout = [dropout] * (len(channel_list) - 1)
+            if plain_last:
+                self.dropout[-1] = 0.
+        else:
+            if len(dropout) is not (len(channel_list) - 1):
+                raise ValueError(
+                    f"Number of dropouts provided ({len(dropout)} does not "
+                    f"match the number of layers specified "
+                    f"({len(channel_list)-1})")
+            self.dropout = dropout
+
         self.act = activation_resolver(act, **(act_kwargs or {}))
         self.act_first = act_first
         self.plain_last = plain_last
 
+        if isinstance(bias, bool):
+            bias_list = [bias] * (len(channel_list) - 1)
+        else:
+            if len(bias) is not (len(channel_list) - 1):
+                raise ValueError(
+                    f"Number of biases provided ({len(bias)})does not match "
+                    f"the number of layers specified ({len(channel_list)-1})")
+            bias_list = bias
```

Comment:
could we also add a simple test case that covers  this new input?

Reply:
Added one in.